### ISSUE TITLE
[Bootcamp] Create layout for Donation entity

### ISSUE THREAD

**Issue by angelina-7** [podkrepi-bg/frontend#511]
Creating dashboard panel for the Donation model

**Comment by Polina1985** [comment]
![Registered user_Donation  profile](https://user-images.githubusercontent.com/77857494/159649726-64ff8e95-8ff6-442c-b974-ff45e937fff8.jpg)

Is this what you have in mind? "Donation layout"? @angelina-7, @IvanovID and @kachar

**Comment by Polina1985** [comment]
![Registered user_Donation  profile](https://user-images.githubusercontent.com/77857494/159649639-f5fca643-61e9-45e8-90b5-95b3b24ee0d1.jpg)

**Comment by angelina-7** [comment]
@Polina1985 That's for the donation grid in the admin panel. I haven't specified I'll make a note for future issues.
![Screenshot 2022-03-24 162855](https://user-images.githubusercontent.com/62725741/159938745-5c400b9f-576a-4117-b05b-8bc241d6625c.jpg)

**Comment by Polina1985** [comment]
Ok, understood. It is connected with the other ticket, where you will add features and I will check again the new features:

'As for the 'delete' option, I consulted with the team and decided to change the status only without removing the record.
Export and print options will be added too, I'll make new issues for them.
Sorting and filtering are part of the grid itself. I've attached a screenshot."
